rpc/documentation/api.md

```diff
@@ -693,6 +694,71 @@ transactions (and no mined transactions).
 
 ___
 
+#### `GetTickets`
+
+The `GetTickets` method queries the wallet for relevant tickets.  The
+query set may be specified using a block range, inclusive, with the heights or
+hashes of the minimum and maximum block.
+
+To avoid exceeding the maximum message size with the return result, a stream is
+used to break up the response into several messages.  A single message will have
+results of a single ticket.
+
+**Request:** `GetTicketsRequest`
+
+- `bytes starting_block_hash`: The block hash of the block to begin including
+  tickets from.  If this field is set to the default, the
+  `starting_block_height` field is used instead.  If changed, the byte array
+  must have length 32 and `starting_block_height` must be zero.
+
+- `sint32 starting_block_height`: The block height to begin including
+  tickets from.  If this field is non-zero, `starting_block_hash` must be
+  set to its default value to avoid ambiguity.  If positive, the field is
+  interpreted as a block height.  If negative, the height is subtracted from the
+  block wallet considers itself in sync with.
+
+- `bytes ending_block_hash`: The block hash of the last block to include
+  tickets from.  If this default is set to the default, the
+  `ending_block_height` field is used instead.  If changed, the byte array must
+  have length 32 and `ending_block_height` must be zero.
+
+- `int32 ending_block_height`: The block height of the last block to include
+  tickets from.  If non-zero, the `ending_block_hash` field must be set to
+  its default value to avoid ambiguity.  If both this field and
+  `ending_block_hash` are set to their default values, no upper block limit is
+  used and transactions through the best block and all unmined transactions are
+  included.
+
+**Response:** `stream GetTicketsResponse`
+
+- `TicketDetails tickets`: A given ticket's details.
+
+  The `TicketDetails` message is used by other methods and is documented
+  [here](#ticketdetails).
```

Comment:
This isn't true, it's a new message that's only used here, and can be documented inline.